利用規約の変更とREADMEの表記修正

R-kawagishi opened the pull request:
@Yoppei 
お疲れ様です。
利用規約の内容とREADMEの表記（特定商取引法の部分）を修正しました。ご確認をお願いいたします。

R-kawagishi committed:
Update README.md

「特定商法取引法」を「特定商取引法」に修正

R-kawagishi committed:
Update terms.html

「2.本サービスの利用」部分に定期課金（サブスクリプション）についてを追記しました。

Yoppei reviewed (approved):
コミット分けていただいてわかりやすかったです。
（GitHubのPRレビュー依頼が完璧すぎて驚いてます。）
ありがとうございます。
修正箇所に不備がなさそうですので、お手隙の際にマージをお願いいたします！

R-kawagishi committed:
Update sct.html

以下修正しました。

・audiotime→東音企画アプリコンテンツの特定商取引法
→audiotimeに特化したものではないので。見えない部分ですが、混乱するので変更しました。
・販売価格の文章
→利用規約に合わせました
・バスティンアプリページ→東音企画アプリコンテンツ
・改訂履歴

R-kawagishi commented:
@Yoppei 
大川先生の教え方が上手だからですね～✨

すみません、特商法の内容も一部変更したのでもう一度ご確認お願いいたします！

Yoppei reviewed (approved):
他の箇所も修正いただいたのですね。
ありがとうございます。

> 以下修正しました。
> 
> ・audiotime→東音企画アプリコンテンツの特定商取引法
> →audiotimeに特化したものではないので。見えない部分ですが、混乱するので変更しました。

🙆‍♂️

> ・販売価格の文章
> →利用規約に合わせました

🙆‍♂️

> ・バスティンアプリページ→東音企画アプリコンテンツ

🙆‍♂️

> ・改訂履歴

🙆‍♂️

R-kawagishi commented:
確認ありがとうございます！
それではマージします！
